lexicons/com/atproto/label/subscribeLabels.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.label.subscribeLabels",
+  "defs": {
+    "main": {
+      "type": "subscription",
+      "description": "Subscribe to label updates",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "cursor": {
+            "type": "integer",
+            "description": "The last known event to backfill from."
+          }
+        }
+      },
+      "message": {
+        "schema": {
+          "type": "object",
+          "required": ["seq", "labels"],
+          "nullable": ["prev"],
+          "properties": {
+            "seq": {"type": "integer"},
+            "labels": {
+              "type": "array",
+              "items": { "type": "ref", "ref": "com.atproto.label.label" }
+            }
+          }
+        }
+      },
+      "infos": [
```

Comment:
ah we've ended up deprecated `infos` as a lexicon/wire-protocol carveout & now they're just in the message schema itself

See subscribeRepos for an example: https://github.com/bluesky-social/atproto/blob/main/lexicons/com/atproto/sync/subscribeRepos.json